src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1152,7 +1152,9 @@ private void UpdateGUIAfterLoadingSpecies(MicrobeSpecies species)
 
     private void CreateMutatedSpeciesCopy(Species species)
     {
-        var newSpecies = CurrentGame.GameWorld.CreateMutatedSpecies(species);
+        Species newSpecies = null;
+        while (newSpecies == null)
+            newSpecies = CurrentGame.GameWorld.CreateMutatedSpecies(species);
```

Comment:
Can this actually return null? Did I miss something. I don't think this change is actually needed?

Reply:
this were some leftovers of old code